source_mesh.py

```diff
@@ -192,14 +188,16 @@ def get_vertex_id(vert_ids, num_phi, num_s, num_theta):
         num_phi (int): number of toroidal angles defining mesh.
         num_s (int): number of closed magnetic flux surfaces defining mesh.
         num_theta (int): number of poloidal angles defining mesh.
+        tor_ext (float): toroidal extent to model (deg).
     
     Returns:
         id (int): vertex index in row-major order as stored by MOAB
     """
     # Compute index offset from magnetic axis
-    if vert_ids[0] == num_phi - 1:
-        # Ensure logical connectivity at final toroidal angle
-        ma_offset = 0
+    if tor_ext == 360.0:
+        if vert_ids[0] == num_phi - 1:
```

Comment:
Should these conditions be combined with an `and`?  right now, `ma_offset` is not determined if `tor_ext == 360` and `vert_ids[0] != num_phi -1`